scripts/functions.js

```diff
@@ -317,15 +317,7 @@ function setFilter(sourceID) {
 		);
 		map.setPaintProperty(
 			sourceID,
-			'circle-opacity',
-			[
-				"interpolate",
-				["linear"],
-				['+', ['to-number', ['get', 'year']], (termLength - 1)],
-				2000, 0.2,
-				(filterStates.year - 1), 0.2,
-				filterStates.year, 1
-			]
+			'circle-opacity', 0.3
```

Comment:
This seems like a great starting point and I will likely experiment with different values